indy_node/server/domain_req_handler.py

```diff
@@ -269,13 +271,30 @@ def _validate_revoc_reg_def(self, req: Request):
         assert revoc_def_tag
         assert revoc_def_type
         tags = cred_def_id.split(":")
+
+        revoc_def = make_state_path_for_revoc_def(tags[0], cred_def_id, revoc_def_type, revoc_def_tag)
```

Comment:
The author DID here is not the author of the corresponding CRED_DEF, but the author of this RevocDef. So, `req.identifier` needs to be passed as the first argument.